airgun/views/repository.py

```diff
@@ -60,16 +60,26 @@ class DockerRepository(View):
         verify_ssl = Checkbox(id='verify_ssl_on_sync')
         upstream_username = TextInput(id='upstream_username')
         upstream_password = TextInput(id='upstream_password')
-        ignore_global_proxy = TextInput(id='ignore_global_proxy')
+        http_proxy_policy = Select(id="http_proxy_policy")
+        proxy_policy = ConditionalSwitchableView(reference='http_proxy_policy')
+
+        @proxy_policy.register('Use specific HTTP Proxy')
+        class HttpProxyForm(View):
+            http_proxy = Select(id="http_proxy")
```

Comment:
That doesn't seem right. Looks like you have support for "Use specific HTTP Proxy" only when creating repository. So there is no support for setting specific proxy when editing repo, and no support for anything except specific proxy when creating repo.

I imagine you should have something like below in all create and edit subclasses:
```python
        @proxy_policy.register(True, default=True)
        class NoSpecificHttpProxy(View):
            pass

        @proxy_policy.register('Use specific HTTP Proxy')
        class HttpProxyForm(View):
            http_proxy = Select(id="http_proxy")
```
Can you try with some existing test cases, or with new small cases (like "create docker repo without setting proxy", "edit docker repo to set specific proxy"; we don't need to merge these in robottelo, they would be just for demonstration)?

Reply:
@mirekdlugosz As per our offline discussion I have filed new issue https://github.com/SatelliteQE/airgun/issues/464 for this.

Reply:
Thanks, Jameer.

Side note: I see there are still no `NoSpecificHttpProxy` blocks for create repositories. I was sure this will cause some of existing tests to fail, but it does not - at least not two I have tried. Apparently, my model on how `ConditionalSwitchableView` works is not correct. I should probably get back to this one day.